event_files/rvv_retired.json

```diff
@@ -0,0 +1,92 @@
+[
+    {
+        "EventName": "RVV_RETIRED.ALL",
+        "EventCode": "0x0",
+        "BriefDescription": "Number of RVV instructions retired",
+        "PublicDescription": "Number of RVV instructions retired"
+    },
+    {
+        "EventName": "RVV_RETIRED.INT",
+        "EventCode": "0x0",
+        "BriefDescription": "Number of integer RVV instructions retired",
+        "PublicDescription": "Number of integer RVV instructions retired"
+    },
+    {
+        "EventName": "RVV_RETIRED.FP",
+        "EventCode": "0x0",
+        "BriefDescription": "Number of floating point RVV instructions retired",
+        "PublicDescription": "Number of floating point RVV instructions retired"
+    },
+    {
+        "EventName": "RVV_RETIRED.ELEMENT.INT8",
+        "EventCode": "0x0",
+        "BriefDescription": "Number of 8-bit integer element operation retired",
+        "PublicDescription": "Number of 8-bit integer element operation retired. For example, if we have SEW=8, LMUL=1, VLEN=128 and doing vector integer arith instruction - it should incremet the RVV_RETIRED.ELEMENT.INT8 counter by 16. Masked-out elements should not increment the counter - so in the previous example if half of the lanes are masked the RVV_RETIRED.ELEMENT.INT8 will be incremented by 8. For multiply-add instructions each element operation should increment counter by 2 to account for both multiplication and addition."
+    },
+    {
+        "EventName": "RVV_RETIRED.ELEMENT.UNMASKED.INT8",
+        "EventCode": "0x0",
+        "BriefDescription": "Number of 8-bit integer element operation retired not taking into account masking",
+        "PublicDescription": "Number of 8-bit integer element operation retired. For example, if we have SEW=8, LMUL=1, VLEN=128 and doing vector integer arith instruction - it should incremet the RVV_RETIRED.ELEMENT.INT8 counter by 16. Mask should not be taken into account - so in the previous example if half of the lanes are masked the RVV_RETIRED.ELEMENT.INT8 will still be incremented by 16. For multiply-add instructions each element operation should increment counter by 2 to account for both multiplication and addition."
```

Comment:
typo: `RVV_RETIRED.ELEMENT.INT8` -> `RVV_RETIRED.ELEMENT.UNMASKED.INT8`, 
the same for `INT16`, `INT32`